netbox/templates/extras/customfieldchoiceset.html

```diff
@@ -0,0 +1,64 @@
+{% extends 'generic/object.html' %}
+{% load helpers %}
+{% load plugins %}
+
+{% block content %}
+  <div class="row mb-3">
+    <div class="col col-md-6">
+      <div class="card">
+        <h5 class="card-header">Custom Field Choice Set</h5>
```

Comment:
Add translate template tags round text.  I'll need to do the one above as it will have merge conflicts.